tests/e2e/test_copy_project.py

```diff
@@ -395,13 +395,13 @@ def test_copy_project_e2e(kili, src_project):
         for asset_src, asset_new in zip(assets_src, assets_new):
             assert asset_src["externalId"] == asset_new["externalId"]
 
-            if Path(asset_src["content"]).is_file():
+            if not asset_src["content"].startswith("http") and Path(asset_src["content"]).is_file():
```

Comment:
can you add a comment to specify that this check is done because of a Python 3.7 pathlib bug ?